### ISSUE TITLE
Get-OMEConfiguration and missing parameters

### ISSUE THREAD

**Issue by KatjaBF** [MicrosoftDocs/office-docs-powershell#1363]
Hi!
The descriptions of the following parameters are missing from Get-OMEConfiguration cmdlet.

IntroductionText
ReadButtonText

PS C:\Users\PowerShell> Get-OMEConfigurationGet-OMEConfiguration
RunspaceId              : 
Image                   : 
ImageUrl                : 
EmailText               : 
PortalText              : 
DisclaimerText          : 
BackgroundColor         : 
IntroductionText        :
ReadButtonText          :
OTPEnabled              :
SocialIdSignIn          : 
ClientEncryptionEnabled : 
ExpirationOptionEnabled : 
Identity                : 
IsValid                 : 
ObjectState             : 

Br Katja

**Comment by chrisda** [comment]
@KatjaBF, the output of a Get- parameter shows the *properties* of an object. If anything, that's a better representation of the parameters available on the corresponding New- or Set- cmdlet, but even then, there are often way more object properties than there are parameters to configure them.

Lots of issues swirling around:

- Typically, we don't document the *properties* that are returned by a Get- cmdlet; the only notable exception is reporting cmdlets.

- I confirmed there are no undocumented *parameters* available on Get-OMEConfiguration.

- I did some related cleanup on Set-OMEConfiguration:

    - I removed the unavailable ClientEncryptionEnabled and ExpirationOptionEnabled cmdlets that were described as 'reserved for internal Microsoft use'.

    - I added the new SocialIdSignIn parameter.

- I've reached out for responsible/knowledgeable people to actually document these new parameters (including IntroductionText and ReadButtonText from your Issue #1354). Feel free to take a crack at the descriptions yourself directly in the topic.

- To get the available parameters on a cmdlet, run this command: `(Get-Command <Cmdlet>).parameters`. Note that in Exchange-related workloads, role based access control (RBAC) can hide otherwise available cmdlets and parameters.

Thanks for the feedback.
